golang.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import "context"
+
+//here is a line coment
+
+// FileGetter is used to retriv the code.
```

Comment:
Typo inside comment in token 'retriv'. Possible corrections:retrieve (0),retain (0),metric (0)